Modules/Filtering/DisplacementField/include/itkTimeVaryingVelocityFieldIntegrationImageFilter.hxx

```diff
@@ -64,11 +64,11 @@ TimeVaryingVelocityFieldIntegrationImageFilter<TTimeVaryingVelocityField,
 {
   const TimeVaryingVelocityFieldType * input = this->GetInput();
   DisplacementFieldType *              output = this->GetOutput();
-  this->m_NumberOfTimePoints = input->GetLargestPossibleRegion().GetSize()[OutputImageDimension];
   if (!input || !output)
   {
     return;
   }
+  this->m_NumberOfTimePoints = input->GetLargestPossibleRegion().GetSize()[OutputImageDimension];
```

Comment:
As you said:
> a slight behaviour change now: `m_NumberOfTimePoints` is no longer set if `input` is provided and `output` is not.

Of course, if you want to be 100% sure that there is no behavior change, you can do, instead:

```cpp
if (input)
{
  this->m_NumberOfTimePoints = input->Get...
} 
if (!input || !output)
{
  return;
}
```

But I agree with you that it probably doesn't matter.